etc/search-headless.api.md

```diff
@@ -11,13 +11,29 @@ export interface AdditionalHttpHeaders {
     'Client-SDK'?: ClientSDKHeaderValues;
 }
 
+// Warning: (ae-forgotten-export) The symbol "Address" needs to be exported by the entry point index.d.ts
+//
+// @public
+export interface AddressDirectAnswer extends BaseFieldValueDirectAnswer<Address> {
+    // Warning: (ae-forgotten-export) The symbol "EnumOrLiteral" needs to be exported by the entry point index.d.ts

```

Comment:
if we export `EnumOrLiteral` from Core, I think we can remove the duplicate model defined in Headless

Reply:
addressed in another pr